spec/provider-spec.coffee

```diff
@@ -294,15 +294,15 @@ describe "HTML autocompletions", ->
     expect(-> completions = getCompletions()).not.toThrow()
     expect(completions[0].displayText).toBe 'onafterprint'
 
-  it "does not provide a descriptionMoreURL if the attribute does not have a unique description", ->
+  it "provide a descriptionMoreURL if the attribute does not have a unique description but starts with on*", ->
```

Comment:
"provide" -> "provides"

Similar tests can also be added now for `aria-` and `role`.